docs/performance/mobile-performance-tuning-1.10-and-earlier.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Tune Mobile Performance (version 1.10 and earlier)
```

Comment:
@natke I didn't figure out how to exclude this page from the sidebar. Ideally we wouldn't show it there. Do you know how to do that?

Reply:
nav_exclude: true